app/sbna-poc/v1_0__initial-form/fields/employment-education.ts

```diff
@@ -220,6 +224,17 @@ export const educationFields: Array<FormWizard.Field> = [
       displayInline: true,
     },
   },
+  {
+    text: 'Give details (optional)',
```

Comment:
would be good to use `detailsFieldWith` to generate this field and add some test coverage for it

Reply:
actually we get the test coverage for free, ignore that

Reply:
Good to know!